cfg_mgmt/bdba.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+import logging
+
+import requests
+
+import cfg_mgmt
+import model.bdba
+
+logger = logging.getLogger(__name__)
+
+
+def get_api_key_expiry(
+    cfg_element: model.bdba.BDBAConfig
```

Comment:
see comment below. Also, I think we should not build such functions around our cfg-model-classes (separation of concerns). It is okay to have a util-function. however, the logic for issueing requests against bdba should be handled by our bdba-client.